spec/requests/v1/role_levels_spec.rb

```diff
@@ -69,10 +73,12 @@
 
   # Test suite for PUT /role_levels/:id
   describe 'PUT /v1/role_levels/:id' do
-    let(:valid_attributes) { { name: 'User' } }
+    let(:valid_attributes) do
+      { name: 'User'}.to_json
+    end
 
     context 'when the record exists' do
-      before { put "/v1/role_levels/#{role_level_id}", params: valid_attributes }
+      before { put "/v1/role_levels/#{role_level_id}", params: valid_attributes, headers: headers }
```

Comment:
May be worth (for all these tests) to test one withouth headers to get a 403. Wdyt?

Reply:
and/or some "read-only" vs "edit" variants

Reply:
@taliaga Before merging this I'll update the test to be a bit more robust, it's probably for the best in the long run.